readme.md

```diff
@@ -26,10 +26,10 @@ The plugin streamlines the entire configuration process and removes the need for
 
 ## Requirements
 
-- Jenkins 2.222.4+
+- Jenkins 2.204.6+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.4+ you can set up an Application Link to have access to all plugin features.
+Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.0+ you can make use of pull request triggers for jobs. With 7.4+ you can set up an Application Link to have access to all plugin features.
```

Comment:
Not fully related maybe. Documentation how pull request feature work is also missing, Yesterday I made a new instance of Jenkins just to see how this works in version 3 of the plugin

Reply:
Thanks for the feedback marcovmun. I will review the readme to see if we can make this process clearer.